dialects/ydb.go

```diff
@@ -0,0 +1,1189 @@
+	case schemas.UnsignedBigInt:
+		yqlType = yql_Uint64
+		return
+	case schemas.Int, schemas.Integer:
+		yqlType = yql_Int32
+		return
+	case schemas.UnsignedInt:
+		yqlType = yql_Uint32
+		return
+	case schemas.Float:
+		yqlType = yql_Float
+		return
+	case schemas.Double:
+		yqlType = yql_Double
+		return
+	case schemas.Blob:
+		yqlType = yql_String
+		return
+	case schemas.Json:
+		yqlType = yql_Json
+		return
+	case schemas.Array:
+		yqlType = yql_List
+		return
+	case schemas.Varchar, schemas.Text:
+		yqlType = yql_Utf8
+		return
+	case schemas.TimeStamp, schemas.DateTime:
+		yqlType = yql_Timestamp
+		return
+	case schemas.Interval:
+		yqlType = yql_Interval
+		return
+	default:
+		yqlType = yql_String
+	}
+	return
+}
+
+func yqlToSQLType(yqlType string) (sqlType schemas.SQLType) {
+	switch yqlType {
+	case yql_Bool:
+		sqlType = schemas.SQLType{Name: schemas.Bool, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Int8:
+		sqlType = schemas.SQLType{Name: schemas.TinyInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Uint8:
+		sqlType = schemas.SQLType{Name: schemas.UnsignedTinyInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Int16:
+		sqlType = schemas.SQLType{Name: schemas.SmallInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Uint16:
+		sqlType = schemas.SQLType{Name: schemas.UnsignedSmallInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Int32:
+		sqlType = schemas.SQLType{Name: schemas.MediumInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Uint32:
+		sqlType = schemas.SQLType{Name: schemas.UnsignedMediumInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Int64:
+		sqlType = schemas.SQLType{Name: schemas.BigInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Uint64:
+		sqlType = schemas.SQLType{Name: schemas.UnsignedBigInt, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Float:
+		sqlType = schemas.SQLType{Name: schemas.Float, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Double:
+		sqlType = schemas.SQLType{Name: schemas.Double, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_String:
+		sqlType = schemas.SQLType{Name: schemas.Blob, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Json:
+		sqlType = schemas.SQLType{Name: schemas.Json, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_List:
+		sqlType = schemas.SQLType{Name: schemas.Array, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Utf8:
+		sqlType = schemas.SQLType{Name: schemas.Varchar, DefaultLength: 255, DefaultLength2: 0}
+		return
+	case yql_Timestamp:
+		sqlType = schemas.SQLType{Name: schemas.TimeStamp, DefaultLength: 0, DefaultLength2: 0}
+		return
+	case yql_Interval:
+		sqlType = schemas.SQLType{Name: schemas.Interval, DefaultLength: 0, DefaultLength2: 0}
+		return
+	default:
+		sqlType = schemas.SQLType{Name: schemas.Blob, DefaultLength: 0, DefaultLength2: 0}
+	}
+	return
+}
+
+func removeOptional(s string) string {
```

Comment:
See `types.IsOptional` for check and get wrapped type

Reply:
To remove function `removeOptional` I think, SDK should be updated so that `func (r *rows) ColumnTypeDatabaseTypeName` and `func (c *conn) GetColumnType` only return wrapped type. What do you think about it?